### PR TITLE
Add skeletal brutes

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

```SUMMARY: Content "Add `skeletal brute` zombies"```

#### Purpose of change

Makes a gap for the evolution between skeleton and skeletal juggernaut.

#### Describe the solution

The `skeletal brute` is an upgrade for the skeleton and upgrades to a skeletal juggernaut by time. They deal moderate cutting damage and performs `"SMASH"` special attack, though their movement speed remains constant.

I've added their spawnrate to be rarer than that of a `zombie brute` in various locations, wherein `skeletal juggernauts`, `zombie brutes` and `zombie hulks` spawn.

I also changed the `skeleton`'s upgrade path to evolve directly into a skeletal brute.

#### Describe alternatives you've considered

These zombies technically has started to generate excess organic armor plating in their body, which results to irregular outgrowths (like the juggernaut's appearance in-game)

#### Additional context

Just tell me what to change for this monster :+1:

### CODE REVIEW

File: data/json/monsters.json [hunk 1]
```diff
@@ -2340,7 +2340,43 @@
...
+    "death_drops": "default_zombie_clothes",
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "HEARS", "BLEED", "HARDTOSHOOT", "REVIVES", "NO_BREATHE", "POISON", "FILTHY" ]
+  },
+  {
+	"//": "Skely brute gaps the evolution between normal and the juggernaut.",
+    "id": "mon_skeleton_brute",
+    "type": "MONSTER",
+    "name": "skeletal brute",
+    "description": "Distorted outgrowths of calcified bone plates covers this zombie's rotten skin. Its joints and cracks around its body oozes with black goo.",
```
Comment (matskuman5) [inline data/json/monsters.json]: Distorted outgrowths of calcified bone plates cover this zombie's rotten skin. Joints and cracks around its body ooze with black goo.

Comment (Axema) [reply data/json/monsters.json]: Done!

**Comment by Firestorm01X2** [comment]
It is better to merge this first:
https://github.com/CleverRaven/Cataclysm-DDA/pull/28797

People starting editing spawn lists freqently with small chunks. My PR edits a lot of them. 

My PR is ready.

File: data/json/monsters.json [hunk 2]
```diff
@@ -2296,7 +2296,43 @@
...
+    "death_drops": "default_zombie_clothes",
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "HEARS", "BLEED", "HARDTOSHOOT", "REVIVES", "NO_BREATHE", "POISON", "FILTHY" ]
+  },
+  {
+	"//": "Skely brute gaps the evolution between normal and the juggernaut.",
+    "id": "mon_skeleton_brute",
+    "type": "MONSTER",
+    "name": "skeletal brute",
+    "description": "Distorted outgrowths of calcified bone plates cover this zombie's rotten skin. Joints and cracks around its body ooze with black goo.",
```
Comment (Night-Pryanik) [inline data/json/monsters.json]:
```suggestion
    "description": "Distorted outgrowths of calcified bone plates cover this zombie's rotten skin.  Joints and cracks around its body ooze with black goo.",
```

**Pull request merged**